ow_lander/scripts/activity_guarded_move.py

```diff
@@ -89,10 +92,18 @@ def pre_guarded_move(move_arm, args):
   plan = move_arm.go(wait=True)
   move_arm.stop()
   move_arm.clear_pose_targets()
+
   print "Done planning approach of guarded_move"
+
   return True
 
-def guarded_move(move_arm, args):
+def guarded_move(move_arm,     # type: class 'moveit_commander.move_group.MoveGroupCommander' 
+                 args):        # type: List[bool, bool, float, float, float, float, float, float, float] 
+
+  print "guarded_move"
+  print type(args)
+  for i in args:
+    print type(i)
```

Comment:
These seem for debugging
```suggestion
```

Reply:
Oh yes, sure, I'll remove those. I thought I removed them all.